### DISCUSSION TITLE
铭瑄H610ITX 2LAN - i3 12100F - rx 560 - AX211 可以使用

### DISCUSSION THREAD

**Discussion by Arjen10** [LimeVista/Hackintosh-H610-12490F-AX201#22] [Show and tell]
经过两天的折腾，整理下遇到的问题

## 卡EB问题（包括issues中也提到了这个问题）

```txt
#LEB.LD.LF|IN] 0 1<"<null string>”><”1”>
#IEB.8.SBSIS21 723512
#LEBIB:SHA1 <5ff7a493bd3c631683fa0d2730af039b85952466>
#LEB.WL.PWLFNV|!] Err(0xE)<- RT.Gy wake-fallure 7C436110-AB2A
#LEB.WL.DTI!] Err(OxE)<- EB.WL.PWLFNV
#IEB.WL.PWLFRTCINONE]
#LEB.WL.DTI!] Err(OxE)<- EB.WL.PWLFRTC
#[EB.LD .LKCIR.1]<"System\Library\KernelCollectionsiBootKernelE)
#LEB.LD.LKFS1-?] 0k(0)
#LEB.LD.LKCI-?1 0k(0)
#LEBIBST:REV11#LEBICSR:IN] 0x00000040#[EBICSR:0UT1 0x00000040
#LEB.BST.FBSI+]
#LEB.BST.FBSIADSZ] 0#IEB .BST.FBSIKSSZ] 0
#[EBISB:SBGMFNS1 j160ap.im4m
#[EB|RH:PF1 usristandalonel0s .dmg.root hash
#[EBIRH:MF]<"usristandalonelt0s.dmg.root hash.j160ap.im4m")
#LEB.LD.LFlIN] 0 1<"usrlistandalonel0s.dmg.root hash"><”0”
#LEB.LD .OFSIOPEN!1 Err(0xE)<"usriistandalonell0s.dmg.root hash”)
#LEB.RH.LRHIP!I Err(OxE)<- EB.LD.LF
#LEB.BST.FBS!!] Err(OxE)<- EB.RH.LRH
#LEBIL0G:DT1 2024-04-13T22:03:42
#[EBILOG:EXITBS:START1 2024-04-13T22:03:42
```

[参考远景解决方案](https://bbs.pcbeta.com/viewthread-1961581-1-1.html)

勾选 Config-booter-Quirks-SetupVirtualMap

## Sonoma 14.4验证

无法加载14.4，现目前使用14.2.1正常使用。没有研究为什么，应该是EFI不适配最新Macos导致。

## 总结

此EFI适配铭瑄H610系列应该问题不大。

1、USB端口也无需二次定制（我只用到了四个USB）。

2、蓝牙、Wi-Fi intel AX211 网卡（作者更新了最新上游最新kext），现目前驱动正常。

3、2.5G网口正常（我是升级版主板，多出的1G网口未做测试，我也不需要，只需要2.5G网口即可）。

感谢此仓库作者，以及大家的问题反馈和对应的解决方案。

**Comment by Arjen10** [comment]
请注意：我使用作者的CPUFriendDataProvider文件使用CMake编译c++项目时，会出现黑屏然后无法开机现象。内存条双通道无法使用，只能使用单通道。

主板返厂铭瑄检查后没有发现问题，到手后，双通道正常，内存条原本支持3200频率，但是实际上使用的是2667，并未内存超频，使用铭瑄bios默认配置。然后我本地生成了CPUFriendDataProvider文件，Geekbench跑分时，我的i3 12100f不能跑满睿频，最多为3.4HZ，然后正常编译c++项目。

**Reply by LimeVista** [reply]
黑屏无法开机可能是内存条兼容问题，铭瑄的主板 BIOS 总是有这种奇奇怪怪的问题。
降低内存时序、调整电压和频率可能可以解决这个问题。
本条经验我已经添加到 `README.md` 方便更多人看到。
